SECURITY_CONTACTS

```diff
@@ -0,0 +1,16 @@
+# Defined below are the security contacts for this repo.
```

Comment:
I wonder if it's worth noting somehow that this only reflects the scripts and tools in this repo (kubernetes/release) and not kubernetes itself.

I guess the all-caps DO NOT REPORT SECURITY VULNERABILITIES comment below might be sufficient.

Reply:
@ixdy this is a straight copy from the template in https://github.com/kubernetes/kubernetes-template-project. It looks fine to me, but if we're going to plan to make a change over there.

Reply:
if you're good with this, do you mind approving?  I don't want to do the lgtm and approve.